src/components/common/TmSessionSignUpPassword.vue

```diff
@@ -111,10 +111,13 @@ export default {
     }
   },
   methods: {
+    prefix() {
```

Comment:
better make this a computed? this way you avoid needing to call a function and can just assume that this value is set.

Reply:
I first set it as a computed value in fact, but strangely enough it didn't work :woman_shrugging: 
Maybe `computed` executes even before that `this.$route` is set?

Reply:
computed executes on every change of the contained values. So you also need to account for the parameter not being set, which is ok as this would default to `undefined` in this case